keylime/ima/ima.py

```diff
@@ -572,6 +576,7 @@ def canonicalize_allowlist(alist_bytes, checksum=""):
         else:
             logger.debug("Allowlist does not specify a version. Assuming current version %s", ALLOWLIST_CURRENT_VERSION)
 
+        alist["meta"]["generator"] = "keylime-allowlist-compatible-format"
```

Comment:
The advantage of using an Enum is that here and in like 36 will be hard to make a typo. And searching over integers will be faster (if this becomes a bottleneck we can create an index, that will fit in memory)